contributors/contributors/community.md

```diff
@@ -14,54 +14,37 @@ The Mattermost community has two parts, Users and Contributors, detailed below.
-  * [Documentation Contributors](https://docs.mattermost.com/process/community-overview.html#documentation-contributors)
-  * [Quality Assurance Contributors](https://docs.mattermost.com/process/community-overview.html#quality-assurance-contributors)
-  * [Knowledge Base Contributors](https://docs.mattermost.com/process/community-overview.html#knowledge-base-contributors)
-  * [Deployment Solution Contributors](https://docs.mattermost.com/process/community-overview.html#deployment-solution-contributors)
-  * [UX Design Contributors](https://docs.mattermost.com/process/community-overview.html#ux-design-contributors)
-  * [Coordination Contributors](https://docs.mattermost.com/process/community-overview.html#coordination-contributors)
-  * [Mattermost Staff](https://docs.mattermost.com/process/community-overview.html#mattermost-staff)
-
-### [Users](https://docs.mattermost.com/process/community-overview.html#contents)
+### Users
 
 In general, all members of the Mattermost community are users of the open source project. They may be end users on a system, administrators, or developers on the Mattermost platform in some way, and the common bond we all share is we benefit from the project’s continued and growing success.
 
 The user community has a number of segments:
 
-#### [End Users](https://docs.mattermost.com/process/community-overview.html#contents)
+#### End Users
 
 People using Mattermost to communicate and achieve their organization’s top priorities.
 
-#### [IT Administrators](https://docs.mattermost.com/process/community-overview.html#contents)
+#### IT Administrators
 
 People deploying Mattermost and supporting end users.
 
-#### [System Integrators](https://docs.mattermost.com/process/community-overview.html#contents)
+#### System Integrators
 
 People who extend and customize Mattermost privately but who do not share back their work with the open source community are considered “users”. When the same people decide to share their work with others in the community they are recognized as “contributors”
 
-#### [Customers](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Customers
 
 Organizations that purchase Mattermost Enterprise Edition as an extension of the Mattermost open source project for advanced features. While the Mattermost roadmap is focused on benefiting the broad community, large customers may accelerate certain parts of the roadmap they find most valuable through large purchases of Enterprise Edition licenses.
 
-### [Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+### Contributors
 
 A subset of the Mattermost community as “contributors” who help increase the overall benefits of Mattermost for others. This may be as simple as offering feature ideas or reporting bugs, offering professional services to install or deploy the system, or it can be as involved as building whole new products based on Mattermost, or contributing code to the underlying software.
```

Comment:
```suggestion
Contributors are a subset of the Mattermost community who help increase the overall benefits of Mattermost for others. This may be as simple as offering feature ideas, reporting bugs, or offering professional installation and deployment services. Or it can be as complex as building whole new products based on Mattermost, or contributing code to the underlying software.
```